### PR TITLE
Blog styles

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
This PR will add styles to blog page

1. code and code block  image
2. List style image
3. Hyperlink color image
4. blockquote  image

### CODE REVIEW

File: website/src/pages/BlogPage/style.ts [hunk 1]
```diff
@@ -165,24 +165,58 @@ const useStyles = makeStyles((theme: Theme) => ({
...
     },
-    '& code':{
-      overflow: 'auto',
+    '& pre code':{
       borderRadius: '4px',
       display: 'block',
       overflowX: 'auto',
       padding: '1em',
-      background: theme.palette.grayishGreen.dark,
+      background: theme.palette.primary.dark,
```
Comment (Pallavi-PH) [inline website/src/pages/BlogPage/style.ts:205]: why this color change?

Comment (Pallavi-PH) [reply website/src/pages/BlogPage/style.ts:205]: The existing color is as per figma screen https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=4%3A6260

File: website/src/pages/BlogPage/style.ts [hunk 2]
```diff
@@ -165,24 +165,58 @@ const useStyles = makeStyles((theme: Theme) => ({
...
       color: theme.palette.grayishGreen.light
     },
+    '& p > code':{
+      padding: theme.spacing(.5,1),
+      borderRadius: '4px',
+      background: theme.palette.blue.light,
```
Comment (Pallavi-PH) [inline website/src/pages/BlogPage/style.ts:211]: Where is this color taken from?

File: website/src/pages/BlogPage/style.ts (hunk 1 above)
Comment (rakeshPRaghu) [reply website/src/pages/BlogPage/style.ts:205]: That's a screenshot,

Comment (rakeshPRaghu) [reply website/src/pages/BlogPage/style.ts:205]: https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=981%3A479 the code block taken from documents design

File: website/src/pages/BlogPage/style.ts (hunk 2 above)
Comment (rakeshPRaghu) [reply website/src/pages/BlogPage/style.ts:211]: from figma only, https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=981%3A479 but from documents design as we dont have any component design for code block in blog

File: website/src/pages/BlogPage/style.ts (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/pages/BlogPage/style.ts:205]: Correct, but I think code block screenshot is added with the intended color theme.

Comment (rakeshPRaghu) [reply website/src/pages/BlogPage/style.ts:205]: i dont know but the same screenshot is present in current blog also, https://openebs.io/blog/openebs-node-device-management-ndm-e2-80-8a--e2-80-8atroubleshooting-tips/

Comment (rakeshPRaghu) [reply website/src/pages/BlogPage/style.ts:205]: i think its better to keep the same style between docs and website

File: website/src/pages/BlogPage/style.ts [hunk 3]
```diff
@@ -165,24 +165,73 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+      background: 'transparent',
+      borderLeft: 'none',
+      textAlign:'center',
+      margin: theme.spacing(3,0),
+      '&::before': {
+        color: theme.palette.info.light,
+        content: 'open-quote',
+        fontSize: '60px',
+        lineHeight: '10px',
+        marginRight: '10px',
```
Comment (Pallavi-PH) [inline website/src/pages/BlogPage/style.ts:230]: theme spacing

File: website/src/pages/BlogPage/style.ts (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/pages/BlogPage/style.ts:205]: Okay

**Review by Pallavi-PH — APPROVED** [review]

**Review by IsAmrish — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**